docs/functions.rst

```diff
@@ -1572,6 +1572,64 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _backupdatausingkopiaserver:
+
+BackupDataUsingKopiaServer
+--------------------------
+
+This function backs up data from a container into any object store
+supported by Kanister using Kopia Repository Server as a data mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that back up the volume and store it on the object store.
+
+   Additionally, in order to use this function, a RepositoryServer CR is required
```

Comment:
```suggestion
   Additionally, in order to use this function, a RepositoryServer CR is required.
```

Reply:
I think we should clearly specify that this repository server CR is needed while creating the actionset. Because, right now it looks like this repo server is expected somewhere in the function args.